### DISCUSSION TITLE
Copy And Paste

### DISCUSSION THREAD

**Discussion by moormast** [pybricks/support#1403] [General]
New to Pybricks and very little/basic knowledge of IT. I want to operate the 4x4 Mercedes Benz Zetros Trial Truck With Powered Up Remote Control. I have paired the Technic Hub and copied the Pybricks code but am unable to pates it. Can you help please

**Comment by laurensvalk** [comment]
I think you might still need to create a new file first:

![image](https://github.com/pybricks/support/assets/12326241/dd06ea1e-0078-4ef8-af3b-80a1d19ec0fc)

**Reply by moormast** [reply]
Created file but when I try and paste receive the message 'Cannot Edit In Read Only Mode@

**Reply by laurensvalk** [reply]
Did you create a new Python file, or a new block coding project?

If you are trying to run Python code, you'll want to create a Python file.

Can you share a screenshot of the message you see?

**Comment by moormast** [comment]
<img width="960" alt="Screenshot 2024-01-29 12 30 04" src="https://github.com/pybricks/support/assets/158049318/d5103c85-e60d-49d3-b944-88fe40dccb95">

**Reply by laurensvalk** [reply]
Yes, looks like you've opened a block project. :slightly_smiling_face: 

If you want to run Python code you found online, just create a new Python project:

![image](https://github.com/pybricks/support/assets/12326241/0ff150ef-e528-4a03-aa4f-033615f0525e)

**Reply by moormast** [reply]
Have just about got there! But I receive this error message. What should I do?
<img width="960" alt="Screenshot 2024-01-30 10 15 25" src="https://github.com/pybricks/support/assets/158049318/c5d9a56c-2bfd-4f2d-bda7-87f3831eae13">

**Reply by laurensvalk** [reply]
Nice!

It says "Timed Out" on line 14, which means it timed out trying to find the remote (which is on line 14).

Try turning on the remote before you start the program, to give your hub the most time to find it.

We're working to improve this part, so the timing won't be as important anymore in the future.

**Reply by moormast** [reply]
All operational! Many Thanks!! One last question. On the YouTube video I have been watching there is an option on the gear wheel to 'include current programme' on the hub so I do not have to use the computer in the future but am unable to locate this option.

**Reply by laurensvalk** [reply]
That is all automatic now. You don't have to do anything.

If you turn the hub off and on again, the program is still there. Just start it with the button :smile:
